example/tests/test_rendering_strategies.py

```diff
@@ -0,0 +1,209 @@
+from __future__ import absolute_import
```

Comment:
this test actually tests the renderer and not the serializer itself. Also one strategy is actually deprecated and will be removed in the future so no need to add an additional test module. Best move this tests to `test_renderers` and see how the other tests are written there so there doesn't need to be a full request/response cycle in the test itself.